orchestrator/kubernetes/kubernetes.go

```diff
@@ -0,0 +1,481 @@
+package kubernetes
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/Sirupsen/logrus"
+	"github.com/pkg/errors"
+
+	"github.com/rancher/longhorn-manager/crdstore"
+	"github.com/rancher/longhorn-manager/orchestrator"
+	"github.com/rancher/longhorn-manager/types"
+	"github.com/rancher/longhorn-manager/util"
+
+	apiv1 "k8s.io/api/core/v1"
+	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	kCli "k8s.io/client-go/kubernetes"
+)
+
+const (
+	cfgDirectory = "/var/lib/rancher/longhorn/"
+	hostUUIDFile = cfgDirectory + ".physical_host_uuid"
+)
+
+var (
+	WaitDeviceTimeout = 30 //seconds
+	WaitAPITimeout    = 30 //seconds
+	WaitPodTimeout    = 2  //seconds
```

Comment:
It's too aggressive. If there is a image download going on, this can easily time out.